igvm/hypervisor.py

```diff
@@ -701,19 +728,24 @@ def migrate_vm(
                     )
             elif offline_transport == 'xfs':
                 self._wait_for_shutdown(vm, no_shutdown, transaction)
-                with target_hypervisor.xfsrestore(vm, transaction) as listener:
+                with target_hypervisor.xfsrestore(
+                    vm=vm, transaction=transaction, vg_name=target_vg_name
```

Comment:
```suggestion
                    vm=vm, transaction=transaction
```
xfsrestore() has no vg_name argument. target_vg_name is also undefined.